### PR TITLE
Move shared ansible constants

### PR DESCRIPTION
**Pull request by didib**

### CODE REVIEW

File: automation/check-patch.sh [hunk 1]
```diff
@@ -29,10 +29,9 @@ export ANSIBLE_CALLBACK_PLUGINS=src/ansible
...
 ANSIBLE_LOG=exported-artifacts/tests/filter-test-ansible-output-$(date -u +%Y%m%d%H%M%S).log
 
-# TODO this doesnn't work, especially since that HE_ANSIBLE_LOG_PATH is never created
-#dnf install -y ansible
-#ansible-playbook --inventory=localhost, -vvvvvv tests/ansible/playbooks/test_filtering.yml > "${ANSIBLE_LOG}" 2>&1
-#if grep 'secret_data' "$HE_ANSIBLE_LOG_PATH"; then
-#	echo Found non-filtered secrets in the log ^^^
-#	exit 1
-#fi
+dnf install -y ansible
```
Comment (mwperina) [inline automation/check-patch.sh:32]: ansible-core?

**Review by mwperina — COMMENTED** [review]
So if ansible callback should be built with Python 3.8 to work with ansible-core >= 2.12, wouldn't it be better to move them into ovirt-hosted-engine-setup-ansible package and build them with python 3.8?

@mnecas is trying to create 3.6/3.8 hybrid build for SDK in https://github.com/oVirt/python-ovirt-engine-sdk4/pull/65

File: automation/check-patch.sh (hunk 1 above)
Comment (didib) [reply automation/check-patch.sh]: Done

**Comment by didib** [comment]
> So if ansible callback should be built with Python 3.8 to work with ansible-core >= 2.12, wouldn't it be better to move them into ovirt-hosted-engine-setup-ansible package and build them with python 3.8?
> 
> @mnecas is trying to create 3.6/3.8 hybrid build for SDK in [oVirt/python-ovirt-engine-sdk4#65](https://github.com/oVirt/python-ovirt-engine-sdk4/pull/65)

That's definitely an option, but current works as-is and is probably enough for our case. This isn't library code per se, like the sdk - "general" code does not import it at random expecting it to be available.

**Comment by michalskrivanek** [comment]
lgtm. manually tested upgrade with full repos enable

**Pull request merged**